boltdesignsystem/bolt
# Create a new component documentation update

## Summary

Updated the create a new component documentation to leverage the 'yarn cc' command.

## Details

Rewrote the create a new component documentation to leverage the streamlined yarn command, this should provide more stability moving forward regrading to keeping up with the documentation.

## How to test

Navigate to https://boltdesignsystem.com/docs/guides/building-a-new-component.html and check to see if the documentation is grammatically and technically correct
